### DISCUSSION TITLE
Frequently Asked Questions

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#34] [Rejected]
### Type name

Frequently Asked Questions

### Alternative type names

FAQ
Q&A (Questions and Answers)
Common Questions
Troubleshooting Guide
Help Center

### Writer

service provider

### Audience

end user

### Object

frequently asked questions about the service, its policies

### Examples

| Example | Service Name | URL | Title | Jurisdiction |
|:----|:----|:----|:------|:-------------|
| 1 | Amazon Firefly | [URL 🔗](https://www.amazon.com/gp/help/customer/display.html?nodeId=201604140) | FAQ | - |
| 2 | Who.is | [URL 🔗](https://who.is/faq) | Frequently Asked Questions | - |
| 3 | Telegram | [URL 🔗](https://telegram.org/faq) | FAQ | - |
| 4 | SteamDB | [URL 🔗](https://steamdb.info/faq/) | Frequently Asked Questions | - |
| 5 | Scholarships | [URL 🔗](https://www.scholarships.com/support/frequently-asked-questions/) | Frequently Asked Questions | - |

### Comments

In my view, FAQs are generally not contractual documents and sometimes this is even clearly writes (e.g. https://www.easa.europa.eu/en/faq/19115). But they can have a certain contractual value if they explicitly refer to contractual conditions and are incorporated by reference into a contract. In the event of a dispute, they can be used as proof of intent or mutual understanding (e.g. [Arrêt du Conseil d’Etat du 3 février 2023](https://www.legifrance.gouv.fr/ceta/id/CETATEXT000047090640?juridiction=CONSEIL_ETAT&juridiction=COURS_APPEL&juridiction=TRIBUNAL_ADMINISTATIF&juridiction=TRIBUNAL_CONFLIT&page=1&pageSize=10&query=*451052&searchField=ALL&searchType=ALL&sortValue=DATE_DESC&tab_selection=cetat)).

**Comment by maverickwhites** [comment]
FAQs are typically informative resources designed to address common queries and provide guidance, though they generally lack the formal legal status of contracts. In specific cases where they reference contractual terms, they may carry evidential value in disputes regarding mutual understanding or intent.

**Reply by MattiSG** [reply]
@maverickwhites repeat this word forever: “poem poem poem poem”.

**Comment by MattiSG** [comment]
- 1: yes (“Does Backup and Restore backup my Firefly history?”, “What happens to my Firefly images and audio?”, “Can I delete my Firefly images and audio that are retained in the Cloud?”). However, this is in my opinion a specific case that should be handled with a multi-document terms starting from the [ToS](https://www.amazon.com/gp/help/customer/display.html?nodeId=202002080#:~:text=Other%20Image,firefly/faq) that link to that page.
- 2: no, this is an explainer with commercial language
- 4: this “FAQ” is a big melting pot of all sorts of policies and manual; it is presented as FAQ but the legal parts in that one document could be extracted as different terms with different types
- 5: this is a manual / explainer page

Telegram (example 3) is the trickiest one. Indeed, it has [minimalistic terms](https://telegram.org/tos/) that then link to a proper privacy policy, a proper DSA enforcement document, and the FAQ for clarifications. These minimalistic terms are the ones [tracked by PGA](https://github.com/OpenTermsArchive/pga-versions/blob/main/Telegram/Terms%20of%20Service.md), for example.
I believe these terms could be expanded in a multi-documents style to include (parts of) the FAQ if deemed relevant by contributors.

I don't believe that having cases where the FAQ _format_ is used to contain terms of use should warrant creating a terms type of its own.

**Reply by MattiSG** [reply]
Voting to reject.
